index.dd

```diff
@@ -98,8 +96,14 @@ void main()
 ----
 )
 
+) $(COMMENT your-code-here)
+))) $(COMMENT intro, div, div)
+
+$(COMMENT
+Picking the frontpage example after the page has loaded leads to reflows
+and re-rendering. Hence, we pick the example as soon as possible
 )
-)))
+<script src="$(ROOT_DIR)js/example_roulette.js"></script>
```

Comment:
I think we need to worry about relayouts first; redraws shouldn't be much of a problem as long as they aren't too distracting. Avoiding relayouts should be easy by reserving space for the interactive stuff that will be filled in later by JS. 

As for redraws (I assume you mean pop-in when new UI elements are displayed), we have too much stuff redrawing anyway with all the widgets etc. Not sure if it's worth complicating the implementation much just to be able to have the drop-down there from the start, though even so, I think it would be like 1 more line to select the right option, right? And the buttons are going to be added in later either way, right?